### PR TITLE
Template tweaks based on review of nf-core/methylseq#108

### PR DESCRIPTION
**Pull request by ewels**
Minor template tweaks, picking up stuff mentioned in nf-core/methylseq#108

### CODE REVIEW

File: nf_core/pipeline-template/{{cookiecutter.name_noslash}}/.gitignore [hunk 1]
```diff
@@ -3,5 +3,6 @@ work/
 data/
 results/
 .DS_Store
-tests/test_data
+tests/
```
Comment (MaxUlysse) [inline nf_core/pipeline-template/{{cookiecutter.name_noslash}}/.gitignore]: could we have test*

Comment (ewels) [reply nf_core/pipeline-template/{{cookiecutter.name_noslash}}/.gitignore]: We _could_, yes.

**Comment by ewels** [comment]
Tests should pass when either https://github.com/nf-core/tools/pull/455 or #457 is merged.

File: nf_core/pipeline-template/{{cookiecutter.name_noslash}}/docs/usage.md [hunk 2]
```diff
@@ -96,7 +96,7 @@ Use this parameter to choose a configuration profile. Profiles can give configur
 
 Several generic profiles are bundled with the pipeline which instruct the pipeline to use software packaged using different methods (Docker, Singularity, Conda) - see below.
 
-The pipeline also dynamically loads configurations from https://github.com/nf-core/configs when it runs, making multiple config profiles for various institutional clusters available at run time. For more information and to see if your system is available in these configs please see the [nf-core/configs documentaiton](https://github.com/nf-core/configs#documentation).
+The pipeline also dynamically loads configurations from [https://github.com/nf-core/configs](https://github.com/nf-core/configs) when it runs, making multiple config profiles for various institutional clusters available at run time. For more information and to see if your system is available in these configs please see the [nf-core/configs documentaiton](https://github.com/nf-core/configs#documentation).
```
Comment (phue) [inline nf_core/pipeline-template/{{cookiecutter.name_noslash}}/docs/usage.md]: [nf-core/configs documentaiton](https://github.com/nf-core/configs#documentation) has a typo :eyes:

File: nf_core/pipeline-template/{{cookiecutter.name_noslash}}/.gitignore (hunk 1 above)
Comment (MaxUlysse) [inline nf_core/pipeline-template/{{cookiecutter.name_noslash}}/.gitignore]:
```suggestion
test*
```

File: nf_core/pipeline-template/{{cookiecutter.name_noslash}}/.gitignore [hunk 3]
```diff
@@ -3,5 +3,6 @@ work/
...
+testing/
```
Comment (MaxUlysse) [inline nf_core/pipeline-template/{{cookiecutter.name_noslash}}/.gitignore]:
```suggestion
```

File: nf_core/pipeline-template/{{cookiecutter.name_noslash}}/docs/usage.md [hunk 4]
```diff
@@ -92,9 +92,16 @@ This version number will be logged in reports when you run the pipeline, so that
 
 ### `-profile`
 
-Use this parameter to choose a configuration profile. Profiles can give configuration presets for different compute environments. Note that multiple profiles can be loaded, for example: `-profile docker` - the order of arguments is important!
+Use this parameter to choose a configuration profile. Profiles can give configuration presets for different compute environments.
 
-If `-profile` is not specified at all the pipeline will be run locally and expects all software to be installed and available on the `PATH`.
+Several generic profiles are bundled with the pipeline which instruct the pipeline to use software packaged using different methods (Docker, Singularity, Conda) - see below.
+
+The pipeline also dynamically loads configurations from [https://github.com/nf-core/configs](https://github.com/nf-core/configs) when it runs, making multiple config profiles for various institutional clusters available at run time. For more information and to see if your system is available in these configs please see the [nf-core/configs documentaiton](https://github.com/nf-core/configs#documentation).
```
Comment (MaxUlysse) [inline nf_core/pipeline-template/{{cookiecutter.name_noslash}}/docs/usage.md]:
```suggestion
The pipeline also dynamically loads configurations from [https://github.com/nf-core/configs](https://github.com/nf-core/configs) when it runs, making multiple config profiles for various institutional clusters available at run time. For more information and to see if your system is available in these configs please see the [nf-core/configs documentation](https://github.com/nf-core/configs#documentation).
```

**Comment by ewels** [comment]
Done!

**Pull request merged**